lib/Idream/Types.hs

```diff
@@ -94,7 +94,7 @@ data Command = Fetch                        -- ^ Fetches all dependencies as des
              | Compile                      -- ^ Compiles all code (fetch needs to be done first to get dependencies code?)
              | Clean                        -- ^ Cleans up build artifacts, fetched code?
              | Run [Argument]               -- ^ Runs the executable defined in idr-project.json
-             | Repl                         -- ^ Opens up the repl
+             | Repl ProjectName PackageName  -- ^ Opens up the repl
```

Comment:
Why did you make it so we have to provide project name and package name?
I haven't messed around with REPL command too much.. but I would imagine mainly loading the main packages we are working on? (the current 'project' we are working on)?

Reply:
Not sure how to determine what's the "main" package.  (Also I marked the PR WIP for Work-In-Progress)